### PR TITLE
a variety of turf/map related fixes, grass resprite

### PR DESCRIPTION
**Pull request by pudl**
-adds the removed sand1, 2, and 3 icons that sand is supposed to break into
-resprites grass to be less weirdly ugly and coder-spritey
-removes grass having borders
-puts green borders around every instance of grass (not in the screenshot, i did it later)
-fixes broken stairs in nuke op base (both meta and box z2)
-fixes broken chairs on metastation
-fixes broken chickens on metastation

### CODE REVIEW

**Comment by pudl** [comment]
map merge didnt work? 1 sec

**Comment by paprka** [comment]
What? So you need to manually put green borders around grass turfs? What happens when people place astroturf?

Take a proper screenshot please, grass needs edging.

File: code/game/turfs/simulated/floor/fancy_floor.dm [hunk 1]
```diff
@@ -59,17 +59,6 @@
 		user << "<span class='notice'>You shovel the grass.</span>"
 		make_plating()
 
-/turf/simulated/floor/fancy/grass/return_siding_icon_state()
```
Comment (paprka) [inline code/game/turfs/simulated/floor/fancy_floor.dm]: Why did you remove this proc even

Comment (pudl) [inline code/game/turfs/simulated/floor/fancy_floor.dm]: it doesn't do anything anymore without siding sprites

**Comment by pudl** [comment]
i'll take screenshot 1 sec

**Comment by WJohn** [comment]
>not hideous grass

:+1: x 1000000

**Comment by pudl** [comment]
http://puu.sh/g8yhc/1be79e0181.png with the map update

**Comment by paprka** [comment]
Bad. It looks really square and tiled, which is not good. Readd siding sprites please, or resprite them.

**Comment by pudl** [comment]
the sprite is the same as the original grass, just recolored. so blame the original spriter

**Comment by paprka** [comment]
The original grass had siding for a reason.

Can you maybe put the grass changes in another PR? This one contains fixes after all

**Comment by pudl** [comment]
i'd like to hear the feedback of @Nienhaus first, since one art director already liked it

**Comment by paprka** [comment]
That's unrelated to the fact that it's a sprite change bundled with fixes though that doesn't actually fix anything.

**Comment by pudl** [comment]
fixed

**Comment by paprka** [comment]
I wasn't talking about the misleading title but that's a start I guess

Just put it in a different PR, it doesn't have anything to do with these fixes

**Comment by Iamgoofball** [comment]
ew, the smoother grass sprite does not blend well with the sharper floor tile sprites

**Comment by paprka** [comment]
This is why the siding is necessary. It should be resprited to brown or dark green but it's good imo

**Comment by Mandurrrh** [comment]
#7911  Both @phil235 and I had already posted pr's for this particular bug fix. Three of us for one outdated tiles is a little much. I already stepped on phils toes.

**Comment by pudl** [comment]
@mandurrh im aware of your pr, the problem is that your dead grass sprite legitimately looks like vomit

**Comment by pudl** [comment]
whoops @mandurrrh

**Comment by Cheridan** [comment]
How many monkeys does it take to screw in a lightbulb

**Comment by Cheridan** [comment]
Meh, there's other fixes in here besides the grass. @Mandurrrh and @phil235 are already on the case with the grass, let them handle it.

**Comment by pudl** [comment]
@cheridan i'm not going to, because both phil and mandurrh are using insultingly bad sprites.

File: code/game/turfs/simulated/floor/fancy_floor.dm (hunk 1 above)
Comment (paprka) [inline code/game/turfs/simulated/floor/fancy_floor.dm]: We added siding icons for a reason, grass looks really ugly without them.

**Comment by Iamgoofball** [comment]
@Cheridan i think too many cooks is more accurate here

**Comment by pudl** [comment]
FUCKING MAP MERGE
